Documentation/BuildAndDeploy.md

```diff
@@ -16,3 +16,51 @@ be changed inside the Visual Studio solution).
 
 Make sure that the "Target SDK Version" dropdown includes the option "10.0.18362.0" - if this is missing,
 [the latest Windows SDK](https://developer.microsoft.com/windows/downloads/windows-10-sdk) needs to be installed.
+
+
+## Building and deploying MRTK to a Windows Mixed Reality Headset
+
+The Windows Mixed Reality (WMR) headset can be used for Universal Windows Platform (UWP) and Standalone builds.  A Standalone build for a WMR headset requires the following extra steps:
```

Comment:
Small note is that XR SDK also supports native WMR in Standalone builds, without needing SteamVR and the WMR plugin. These steps are required for Unity's legacy XR though.
Not sure if we want to mention this somewhere or not.

Reply:
We can mention it here, I added a note